CONTRIBUTING.md

```diff
@@ -3,53 +3,45 @@
 The development flow includes:
 
 - [Contributing](#contributing)
-  - [Create workspace, clone sources](#create-workspace-clone-sources)
+  - [Start Workspace](#start-workspace)
   - [Build](#build)
   - [Run tests](#run-tests)
   - [Run](#run)
-  - [Generate Devfile](#generate-devfile)
-  - [Create Workspace](#create-workspace)
   - [Package binaries](#package-binaries)
   - [Push changes, provide Pull Request](#push-changes-provide-pull-request)
   - [Make Release](#make-release)
 
-## Create workspace, clone sources
+## Start Workspace
 
-To create a workpace you can use [devfile](devfile.yaml):
+To start a workpace from the [devfile](devfile.yaml) create a workspace URL by appending the chectl repo to the URL of your Che host. For example, to create a workspace in the openshift sandbox use the URL:
 
 ```
-$ chectl workspace:start -f https://raw.githubusercontent.com/che-incubator/chectl/main/devfile.yaml
+https://devspaces.apps.sandbox-m2.ll9k.p1.openshiftapps.com/f?url=https://github.com/che-incubator/chectl
 ```
 
-> See more about [devfile](https://redhat-developer.github.io/devfile)
+> See more about [devfile](https://devfile.io/docs/2.1.0/what-is-a-devfile)
+> See more about [workspace URLs](https://www.eclipse.org/che/docs/stable/end-user-guide/starting-a-new-workspace-with-a-clone-of-a-git-repository/)
 
 After starting the workspace Theia will clone sources of `chectl` to `/projects/chectl` directory.
```

Comment:
```suggestion
After starting the workspace, Theia will clone sources of `chectl` to the `/projects/chectl` directory.
```

Reply:
Since we will soon be switching IDE default from Theia to VSCode, perhaps "After starting the workspace, the IDE will clone sources..." is better?